qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -807,6 +807,34 @@
           {
             "field": "total_in_use_beds_covid",
             "name": "IL Total In Use COVID-19 Beds"
+          },
+          {
+            "field": "npi",
+            "name": "National Provider Identifier"
+          },
+          {
+            "field": "provider_business_practice_city",
+            "name": "Provider Business Practice City"
```

Comment:
this is redundant and can be removed, guppy will auto captilaized 1st letter in each word divided by `_`

you only need to use this section to customize fied name if above rule cannot satisfy your needs